Entropy.h

```diff
@@ -59,7 +59,7 @@ class EntropyCalculator {
         for (auto it = freq.cbegin(); it != freq.cend(); ++it) {
             if (it->second == 0) continue;
             double relativeFrequency(((double)it->second) / numItems);
-            entropy -= relativeFrequency * log2(relativeFrequency);
+            entropy -= relativeFrequency * log_2(relativeFrequency);
```

Comment:
nah this needs to be double precision. The code was never correct in the first place :grimacing: